Intel_SighPP_CMake/Intel_SighPP_CMake/src/ml_lib/caffe_impl.cpp

```diff
@@ -24,29 +26,32 @@ struct CaffeModelImpl : public ModelInterface {
 	const float inScaleFactor = 0.007843f;
 	const float meanVal = 127.5;
 
-	const float confidence_threshold = 0.8f;
+	const float confidence_threshold = 0.9f;
```

Comment:
You prefer to choose higher confidence

Reply:
I only did that because it was recognising cups everywhere XD . Om not sure. We can talk about it tomorrow.